src/app/core/entity/entity-actions/entity-actions.service.ts

```diff
@@ -102,9 +104,22 @@ export class EntityActionsService {
       $localize`:Entity action progress dialog:Processing ...`,
     );
     let result = new CascadingActionResult();
+
     for (let entity of entities) {
+      if ("User" === entity.getType()) {
+        this.keycloakAuthService.deleteUser(entity.getId()).subscribe({
+          next: () => {},
+          error: async () =>
+            await this.confirmationDialog.getConfirmation(
+              $localize`:delete account in keycloak related error title:Keycloak User could not be deleted`,
+              $localize`:delete account in keycloak related error dialog:User Account could not be deleted in Keycloak. Please delete user manually in Keycloak.`,
```

Comment:
What happens if there actually is no user to be deleted? Do we get feedback from the API?

Reply:
Nothing, it's save to call this endpoint multiple times. If something there, we delete it, if not, nothing to delete - also fine.